### ISSUE TITLE
aws emr create-cluster configurations format vs emr dashboard

### ISSUE THREAD

**Issue by d136o** [aws/aws-cli#1502]
The options passed via the json file specified with the --configurations option need to be one of:

Classification, Configurations, Properties

consistent with the capitalization above. This is true of versions 1.7.47 and 1.8.3.

However, if you try to launch the same cluster via the EMR dashboard you will not be able to launch unless the words are lower cased.

This is using EMR 4.0.0 and attempting to configure a spark cluster with the following options:

[{"classification":"spark","properties":{"maximizeResourceAllocation":"true"}},{"classification":"mapred-site","properties":{"mapred.output.direct.EmrFileSystem":"true","mapred.output.direct.NativeS3FileSystem":"true"}}]

In other words:

+ options must be lowercased when passed via text box in the dashboard
+ options must be capitalized when passed via json file when using aws cli

The two seem inconsistent.

**Comment by rayluo** [comment]
What you described about the CLI behavior is true, and the capitalization complies with [the example documented here](http://docs.aws.amazon.com/cli/latest/reference/emr/create-cluster.html#examples) and [here](http://docs.aws.amazon.com/ElasticMapReduce/latest/ReleaseGuide/emr-configure-apps.html). I would like to suggest you to bring this question up in [Amazon EMR Forum](https://forums.aws.amazon.com/forum.jspa?forumID=52). I believe there are more EMR users and experts there who can help you more efficiently.
